HarpGuru/HarpFace/HarpCell/components/index.test.tsx

```diff
@@ -21,19 +21,25 @@ test('A component is rendered with the Degree or Pitch value in its text view de
       return [inactiveCellsHarpFaceProps.activeHarpStrata]
     if (stateItem === 'activeExperienceMode') return [ExperienceModes.Explore]
     if (stateItem === 'quizQuestion') return [DegreeIds.Root]
+    if (stateItem === 'activeDisplayMode') return [DisplayModes.Degree]
     return undefined
   })
   const harpCellProps = {
-    activeDisplayMode: DisplayModes.Degree,
     yxCoord: [3, 0] as [3, 0],
   }
   const { getByText, rerender } = render(<HarpCell {...harpCellProps} />)
 
   expect(getByText(DegreeIds.Second)).toBeTruthy()
+  mockUseGlobal.mockImplementation((stateItem: string) => {
+    if (stateItem === 'activeHarpStrata')
+      return [inactiveCellsHarpFaceProps.activeHarpStrata]
+    if (stateItem === 'activeExperienceMode') return [ExperienceModes.Explore]
+    if (stateItem === 'quizQuestion') return [DegreeIds.Root]
+    if (stateItem === 'activeDisplayMode') return [DisplayModes.Pitch]
+    return undefined
+  })
```

Comment:
These mock implementations are quite ugly and probably larger than required in some places.

Reply:
Referenced in an open issue: https://github.com/js-jslog/harpnative/issues/28#issuecomment-663617750